yt/yt/tests/integration/node/test_layers.py

```diff
@@ -557,6 +563,58 @@ def test_wrong_tag_document_type(self):
         with raises_yt_error(f'Tags document "{TestDockerImage.TAG_DOCUMENT_PATH}" is not a map'):
             self.run_map(f"{TestDockerImage.IMAGE}:tag")
 
+    @authors("khlebnikov")
+    @pytest.mark.timeout(180)
+    def test_invalid_external_image(self):
+        with raises_yt_error('External docker image is not supported in Porto job environment'):
+            self.run_map(self.INVALID_EXTERNAL_IMAGE)
+
+
+@authors("khlebnikov")
+class TestCriDockerImage(TestLayersBase):
+    JOB_ENVIRONMENT_TYPE = "cri"
+
+    INPUT_TABLE = "//tmp/input_table"
+    OUTPUT_TABLE = "//tmp/output_table"
+    MAP_COMMAND = "cat"
+
+    def create_tables(self):
+        create("table", self.INPUT_TABLE)
+        create("table", self.OUTPUT_TABLE)
+
+        write_table(self.INPUT_TABLE, [{"a": 1}])
+
+
+    def run_map(self, **kwargs):
```

Comment:
Did all the tests pass?
```
[fail] test_layers.py::py3_flake8 [default-linux-x86_64-debug] (0.00s) yt/yt/tests/integration/node/test_layers.py:588: [E303]
too many blank lines (2)
```

Reply:
OS ya make sees no problems here

`khlebnikov@nebius-yt-dev:~/src/ytsaurus$ ./ya make --style -L yt/yt/tests/integration/node/
------- [TL] $(B)/list_result.log
Number of suites skipped by suite type: 3

yt/yt/tests/integration/node <py3_flake8>
  test_disk_quota.py::py3_flake8
  test_dynamic_cpu_reclaim.py::py3_flake8
  test_gpu_layers.py::py3_flake8
  test_heartbeats_to_ca.py::py3_flake8
  test_hot_swap.py::py3_flake8
  test_io_engine.py::py3_flake8
  test_job_proxy.py::py3_flake8
  test_jobs.py::py3_flake8
  test_layers.py::py3_flake8
  test_node_discovery_manager.py::py3_flake8
  test_node_dynamic_config.py::py3_flake8
  test_node_flavors.py::py3_flake8
  test_node_job_resource_manager.py::py3_flake8
  test_node_orchid.py::py3_flake8
  test_p2p.py::py3_flake8
  test_user_job.py::py3_flake8

Total 1 suite
Total 16 tests
Ok
khlebnikov@nebius-yt-dev:~/src/ytsaurus$ ./ya make --style  yt/yt/tests/integration/node/
Ok
`

Reply:
What's the result here? Is there any discrepancy between OSS ya and Yandex ya style check behavior?